### ISSUE TITLE
Get rid of the DSL2 declaration in the corresponding template files

### ISSUE THREAD

**Issue by JoseEspinosa** [nf-core/tools#1535]
### Description of feature

From Nextflow version 22.4.0, the DSL2 declaration is not needed anymore at the top of 4.0 script (i.e. nextflow.enable.dsl = 2). See [here](https://www.nextflow.io/blog/2022/evolution-of-nextflow-runtime.html). 
Hence, it can be removed from:
[ ] the `main.nf`of the tests files of the module's template
[ ] the `main.nf`of the pipeline template (not until the default version of nextflow is bumped to 22.4.0)

**Comment by ewels** [comment]
But it doesn't do any harm, right? I'm not in a rush to remove this as it'll break the pipeline for people running older versions of Nextflow, when they might work otherwise..

**Comment by JoseEspinosa** [comment]
Yep, it does no harm, even ask Paolo if it was going to raise any warning and he told me that it was not. Just open the issue to keep track of it in case we want to update the template in the future.

**Comment by mirpedrol** [comment]
Done with https://github.com/nf-core/tools/pull/3046/files
